docker/known_hosts.sh

```diff
@@ -4,17 +4,53 @@ set -eu
 
 known_hosts_file=${1}
 known_hosts_file=${known_hosts_file:-/etc/ssh/ssh_known_hosts}
+hosts="github.com gitlab.com bitbucket.org ssh.dev.azure.com vs-ssh.visualstudio.com"
 
 retries=10
 count=0
 ok=false
+wait=2
 until ${ok}; do
-    ssh-keyscan github.com gitlab.com bitbucket.org ssh.dev.azure.com vs-ssh.visualstudio.com >> ${known_hosts_file} && \
-    sh /home/flux/verify_known_hosts.sh ${known_hosts_file} && ok=true || ok=false
-    sleep 2
+    ssh-keyscan ${hosts} > ${known_hosts_file} && ok=true || ok=false
```

Comment:
This will not help the CI from failing, the actual issue is that `ssh-keyscan` does not report failure when it times out on a host; it will instead just not append anything for that host to the `known_hosts_file`, which results in the verification below failing because some fingerprints are missing.

Reply:
Never mind, I was reviewing a subset.